File: iniciante/java/1013/1013.java
//QUESTÃO 1012 - O Maior

import java.io.IOException;
import java.util.Scanner;
 
public class Main {
	public static void main(String[] args) throws IOException {
	        Scanner sc = new Scanner(System.in);
			            
	        int a = sc.nextInt();
	        int b = sc.nextInt();
 	        int c = sc.nextInt();
						            
 	        int maiorAB = (a + b + Math.abs(a - b)) / 2;
	        int maior = (maiorAB + c + Math.abs(maiorAB - c)) / 2;
								        
		System.out.println(maior + " eh o maior");
	}
}

File: iniciante/c99/1013/1013.c
#include <stdio.h>
#include <stdlib.h>

int main() {

	int a, b, c;

	scanf("%d", &a);
	scanf("%d", &b);
	scanf("%d", &c);

	int maiorAB = (a + b + abs(a - b)) / 2;
	int maior = (maiorAB + c + abs(maiorAB - c)) / 2;

	printf("%d eh o maior\n", maior);

	return 0;

}
